strategy-explorer/strategy-explorer-demo.html

```diff
@@ -29,7 +29,7 @@
                 await p.generate(),
                 await p.generate(),
                 await p.generate(),
-                await p.generate()].map(strategyExplorer.preparePopulation);
+                await p.generate()].map(pop => strategyExplorer.preparePopulation(pop));
```

Comment:
sorry for the stupid question - how are the 2 different?

Reply:
This version binds preparePopulation to strategyExplorer on invocation, so that inside the function the 'this' pointer resolves correctly.

It didn't matter before because we weren't directly referring to local data in preparePopulation.